docs/evaluate/development-production-features/temporal-nexus.mdx

```diff
@@ -17,32 +17,96 @@ keywords:
+
+- **Share a subset of a Temporal Application** \-
+Abstract and share a subset of an Application as a Nexus Service.
+Expose Services on a Nexus Endpoint for others to use and secure access with Endpoint access control policies.
+Nexus Endpoints decouple callers from handlers, so teams can operate more autonomously.
 
-For a deep dive into Temporal Nexus, see the [Temporal Nexus Encyclopedia section](https://docs.temporal.io/nexus).
-To connect with the Nexus community, join the [#nexus](https://temporalio.slack.com/archives/C07LQN0JK9B) channel in [Temporal Slack](https://t.mp/slack).
+- **Design for growth** \-
+Temporal Nexus enables a modular application design that can evolve as you grow.
+Start with Nexus Services in a monolithic Namespace and move Services to separate Namespaces with small configuration changes and no downtime for service consumers.
 
-Temporal keeps the runtime benefits of an Event-Driven Architecture (EDA) where services can be down without affecting each other, but offers a higher-level durable execution programming model that lets you write idiomatic code in any language and preserve the state of execution.
-This ensures that your Workflows can be resumed automatically in the presence of failure, without the extensive boilerplate code and complexity that comes with using raw EDA frameworks directly.
+- **Smaller failure domains** \- When teams operate in the same monolithic Namespace, everything is available to everyone, and mis-behaving workers can trigger rate limits that affect all teams operating in that monolithic Namespace.
+  Nexus enables each team to have their own Namespace for improved security, troubleshooting, and fault isolation.
 
-With Temporal Nexus, inter-service communication and durable execution can span teams, Namespaces, and regions all from within the Temporal context.
-Nexus preserves a familiar API programming model that supports both synchronous and asynchronous patterns, is suitable for low-latency and long-running operations, and uses an optimized form of the Temporal queue-based Worker architecture to preserve the benefits of EDA at runtime.
+- **Multi-region use cases** \-
+Nexus requests in Temporal Cloud are routed across a global mTLS-secured Envoy mesh.
+Reliable execution is provided with built-in Nexus Machinery and Temporal's queue-based Worker architecture.
 
-Temporal Nexus provides an integrated Temporal SDK experience, built-in Nexus Machinery, first-class observability, and enables each team to have their own Namespace for improved security, troubleshooting, and blast radius isolation.
-It allows multiple teams to collaborate autonomously while maintaining a clean separation of concerns, ensuring that applications are scalable, secure, and easy to manage as they grow.
+**Advantages of Temporal Nexus:**
 
-With Temporal Nexus, teams are able to:
+- Reliably connect Temporal Applications within and across Namespaces.
+- Service contracts suitable for sharing across teams.
```

Comment:
Is it Service contracts or API contracts here? Which is more industry standard? Because Nexus modularity provides isolation of responsibility with well defined API surface endpoints

Reply:
thanks for pointing this out. One key goal for this docs updates is to strike a better balance between API/RPC vs. EDA audiences, so moving to a more distinct, but sufficiently general, "microservice contract" -- at least for the evaluate page.